interface/forms/eye_mag/view.php

```diff
@@ -186,10 +214,23 @@
         $output_priors = priors_select("ALL", $id, $id, $pid);
 
         menu_overhaul_left($pid, $encounter);
+        //define if this is a new or est patients for coding auto-suggestions
+            //TODO: develop logic to recognize post-op visits 99024
+            // if a prior encounter within 90 days are procedures with a global period still in effect, then post-op code
         ?>
-
-
-        <!-- start form -->
+          <script>
+              var Code_new_est ='<?php
+                 if  ($output_priors =='') {
+                     echo "New";
+                 } else {
+                     echo "Est";
```

Comment:
translate labels

Reply:
submitting this as 
echo xla("New");

This will end up as javascript.  Does it need one of these new js_escape functions too?

Reply:
No, wouldn't worry about using it here and just use xls(). Can only use the js escaping functions at the endpoint since it places quotes around it in js context.

Reply:
Still need to translate these labels

Reply:
above comment not yet addressed